codalab/worker_manager/azure_batch_worker_manager.py

```diff
@@ -0,0 +1,157 @@
+try:
+    from azure.batch.batch_auth import SharedKeyCredentials  # type: ignore
+    from azure.batch._batch_service_client import BatchServiceClient  # type: ignore
+    from azure.batch.models import (  # type: ignore
+        OutputFile,
+        OutputFileBlobContainerDestination,
+        OutputFileDestination,
+        OutputFileUploadCondition,
+        OutputFileUploadOptions,
+        TaskAddParameter,
+        TaskCounts,
+        TaskContainerSettings,
+    )
+except ModuleNotFoundError:
+    raise ModuleNotFoundError(
+        "Running the worker manager requires the azure-batch module.\n"
+        "Please run: pip install azure-batch==9.0.0"
```

Comment:
do we need the version pin?

Reply:
It looks like they release a new version every 6 months. I don't think it needs to be pinned. I will remove the version for now.